packages/webviz-core/package-lock.json

```diff
@@ -50,11 +50,6 @@
         }
       }
     },
-    "@cruise-automation/hooks": {
-      "version": "0.0.1",
-      "resolved": "https://registry.npmjs.org/@cruise-automation/hooks/-/hooks-0.0.1.tgz",
-      "integrity": "sha512-sUwYShqRJAyg2XS7+uc9ZwdNkRArLdyjPTF6Ebl1GphXHX5QYU14aMt3SO3QtVg0fd/KSWM4FinRosxZDq5uZg=="
-    },
```

Comment:
FYI these changes should not be committed to source control, since you haven't changed anything in `package.json`.

Reply:
I just reverted the changes to package-lock.json in a new commit